src/mpid/ch4/netmod/ofi/ofi_iovec_util.h

```diff
@@ -170,6 +170,8 @@ MPL_STATIC_INLINE_PREFIX void MPIDI_OFI_init_seg_state(MPIDI_OFI_seg_state_t * s
                                                        const MPI_Aint target,
                                                        size_t origin_count,
                                                        size_t target_count,
+                                                       size_t origin_bytes,
```

Comment:
Please add a comment to explain these parameters.
Especially, I think it is important to note the difference between `*_counts` and `*_bytes` here. (although it's a kind of standard in MPICH code, it still wouldn't hurt to clarify)
Also, people may wonder later why there are two similar parameters, `count` and `bytes`, so that should be clearly explained.

Reply:
@huansong Can you also add a comment on *why* both `count` and `bytes` are needed? Something similar to a commit message is fine. I just want to make sure the same information as commit message is also captured in a comment, because people don't always necessarily dig into git history to understand the code.

Reply:
@hajimefu I see. Sorry that I didn't capture the point. I have added the explanation comments.